### ISSUE TITLE
Can I Use This Package Without a License?

### ISSUE THREAD

**Issue by STRockefeller** [syncfusion/flutter-widgets#1242]
Hello,

I would like to address a question regarding the usage of the Syncfusion DataGrid package in my repository. Firstly, I must say that the Syncfusion DataGrid is an excellent package. However, I am confused about whether I need a license to use this package in my repository.

While browsing the [pub.dev page](https://pub.dev/packages/syncfusion_flutter_datagrid) for this package, I came across a disclaimer stating that a license is required. Here is the disclaimer I found:

> "Disclaimer: This is a commercial package. To use this package, you need to have either a Syncfusion commercial license or Free Syncfusion Community license. For more details, please check the LICENSE file."

On the other hand, I also found an article in the [official documentation](https://help.syncfusion.com/flutter/licensing/overview) that claims a license is not necessary. According to the documentation:

> "The License key registration is no longer required for Flutter from version 18.3.0.x. So, there is no need to generate or register Syncfusion Flutter license keys in your Flutter projects. The Flutter controls from Syncfusion can be used without registering the license keys."

Could someone please clarify this matter for me? I would greatly appreciate any information or guidance regarding the licensing requirements for using the Syncfusion DataGrid package.

Thank you for your assistance.

**Comment by annaisdevk** [comment]
Hi,

on the page of your widgets, in the license section, it says:

> Syncfusion License
> 
> Syncfusion Flutter DataGrid package is available under the Syncfusion Essential Studio program, and can be licensed either under the Syncfusion Community License Program or the Syncfusion commercial license.
> 
> To be qualified for the Syncfusion Community License Program you must have a gross revenue of less than one (1) million U.S. dollars ($1,000,000.00 USD) per year and have less than five (5) developers in your organization, and agree to be bound by Syncfusion’s terms and conditions.
> 
> Customers who do not qualify for the community license can contact sales@syncfusion.com for commercial licensing options.
> 
> Under no circumstances can you use this product without (1) either a Community License or a commercial license and (2) without agreeing and abiding by Syncfusion’s license containing all terms and conditions.
> 
> The Syncfusion license that contains the terms and conditions can be found at
> https://www.syncfusion.com/content/downloads/syncfusion_license.pdf

If you meet the requirements for the **community license**, then you can claim from this page: https://www.syncfusion.com/sales/communitylicense

**Comment by STRockefeller** [comment]
@annaisdevk Based on your statement, do I still require a license even if I am using a package version higher than 18.3.0.x? [This page](https://help.syncfusion.com/flutter/licensing/overview) really confuse me.

**Comment by annaisdevk** [comment]
@STRockefeller Yes, it is a bit confusing because all Syncfusion widgets on https://pub.dev show this licence (the one I wrote in the other message). 

What I understand:

1- Syncfusion Flutter widgets can be used without registering licence keys.
2- However, it is necessary to have a Syncfusion Community Licence Program or the Syncfusion Commercial Licence. 

I have requested for a Syncfusion Community Licence, which is free if you meet the requirements. (from [here](https://www.syncfusion.com/sales/communitylicense)) Now I am waiting to see if Syncfusion will grant it to me or not.
